apps/nxls-e2e/project.json

```diff
@@ -5,16 +5,7 @@
   "projectType": "application",
   "targets": {
     "test": {
-      "command": "jest --detectOpenHandles --runInBand --verbose",
-      "options": {
-        "cwd": "apps/nxls-e2e"
-      },
-      "dependsOn": [
-        {
-          "projects": "nxls",
-          "target": "build"
-        }
-      ]
+      "executor": "nx:noop"
```

Comment:
This shouldn't be needed. I can see you already have the plugin configured to infer an `e2e` target and not a `test` target. We should be able to delete the `test` target here.